language-detection-fasttext/src/main/java/com/hpe/caf/languagedetection/fasttext/FastTextScriptExecutor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2015-2019 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.languagedetection.fasttext;
+
+import jep.Jep;
+import jep.JepException;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import jep.NDArray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Uses Jep to call the FastText library in python 3 for language detection.
+ * 
+ */
+public final class FastTextScriptExecutor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(FastTextScriptExecutor.class);
+    
+    /**
+     * An instance of the Jep library with imported FastText  library and loaded model. 
+     * FastText is using Numpy Python Library and closing the Jep instance breaks the library <a href=" https://github.com/mrj0/jep/issues/28">.
+     * Try to reimport it will fail and as a result of this issue FastText can be imported only once.
+     * Jep will only execute calls on the thread it was instantiated on therefore is Jep instatance required here to be static and/or 
+     * thread local, more at <a href="https://github.com/mrj0/jep/wiki/Performance-Considerations">.
+     */
+    private static final Jep JEP = initJep();
```

Comment:
This is different from the ContentSegregation implementation, read the comment there.
https://github.com/CAFDataProcessing/util-email-content-segregation/blob/develop/src/main/java/com/hpe/caf/worker/emailsegregation/ContentSegregation.java#L41

Reply:
I had at first done this implementation similarly to the one at ContentSegregation.java, but then realized it is not necessary and in the end can be done simply by the way I did, following the same conditions. Still want me to change back?

Reply:
The worker is multi-threaded, is it safe to have multiple threads sharing a single Jep object?
For example I see in the detect method JEP.set("text", text) if two threads do this concurrently will one override the others value of text?

Reply:
"The worker is multi-threaded", the information I had wrong here. I remembered we are using single thread per instance, so that's why my decision here. I will update this, thanks.